<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloatConverterWidget</class>
 <widget class="QWidget" name="FloatConverterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>914</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="CustomLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="step2Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="step0Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="step1Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="step4DisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="3">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="startPushButton">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPushButton">
      <property name="text">
       <string>Next Step</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resultPushButton">
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="difficultyComboBox">
     <item>
      <property name="text">
       <string>Easy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Moderate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hard</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="step6Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QRadioButton" name="hexToDecButton">
     <property name="text">
      <string>Hex -&gt; Binary -&gt; Decimal</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="step1DisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="step3DisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="difficultyLabel">
     <property name="text">
      <string>Difficulty</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="step5Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="step9Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="step4Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="customLineEdit"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="exerciseDisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QLabel" name="step7DisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="step3Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="decToHexButton">
     <property name="text">
      <string>Decimal -&gt; Binary -&gt; Hex</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="step5DisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLabel" name="step6DisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="step2DisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="step8Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="step7Label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="exerciseLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="step0DisplayLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
